shims/hdi35/common/src/main/java/org/pentaho/hadoop/shim/hdi35/HadoopShim.java

```diff
@@ -22,51 +22,55 @@
 
 package org.pentaho.hadoop.shim.hdi35;
 
+import org.pentaho.di.core.database.DatabaseInterface;
 import org.pentaho.di.core.exception.KettlePluginException;
+import org.pentaho.di.core.plugins.DatabasePluginType;
+import org.pentaho.di.core.plugins.Plugin;
+import org.pentaho.di.core.plugins.PluginInterface;
+import org.pentaho.di.core.plugins.PluginRegistry;
 import org.pentaho.hadoop.shim.HadoopConfiguration;
 import org.pentaho.hadoop.shim.HadoopConfigurationFileSystemManager;
-import org.pentaho.hadoop.shim.common.DriverProxyInvocationChain;
 import org.pentaho.hadoop.shim.common.HadoopShimImpl;
-import org.pentaho.hadoop.shim.hdi35.invocationhandler.HDIDriverInvocationHandler;
 
-import java.sql.Driver;
+import java.net.URLClassLoader;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Properties;
 
 public class HadoopShim extends HadoopShimImpl {
+
+  public HadoopShim() {
+    super();
+  }
+
   @Override
   public void onLoad( HadoopConfiguration config, HadoopConfigurationFileSystemManager fsm ) throws Exception {
     registerExtraDatabaseTypes( config.getConfigProperties() );
     super.onLoad( config, fsm );
   }
 
   protected void registerExtraDatabaseTypes( Properties configuration ) throws KettlePluginException {
-    String sparkSqlSimbaDriverName =
-      configuration.getProperty( "sparksql.simba.driver", "com.simba.spark.jdbc41.Driver" );
-    JDBC_POSSIBLE_DRIVER_MAP.put( "SparkSqlSimba", sparkSqlSimbaDriverName );
-  }
+    /*String hiveSimbaDriverName = configuration.getProperty( "hive2.simba.driver", "com.simba.hive.jdbc41.HS2Driver" );
+      JDBC_POSSIBLE_DRIVER_MAP.put( "hive2Simba", hiveSimbaDriverName );*/
 
-  @Override
-  public Driver getJdbcDriver( String driverType ) {
-    try {
-      Driver newInstance = null;
-      Class<? extends Driver> clazz = JDBC_DRIVER_MAP.get( driverType );
-      if ( clazz != null ) {
-        newInstance = clazz.newInstance();
-        return DriverProxyInvocationChain.getProxy( Driver.class, newInstance, HDIDriverInvocationHandler.class );
-      } else {
-        clazz = tryToLoadDriver( JDBC_POSSIBLE_DRIVER_MAP.get( driverType ) );
-        if ( clazz != null ) {
-          newInstance = clazz.newInstance();
-          if ( driverType.equals( "Impala" ) ) {
-            return DriverProxyInvocationChain.getProxy( Driver.class, newInstance, HDIDriverInvocationHandler.class );
-          }
-          return newInstance;
-        }
-        return null;
-      }
+    String impalaSimbaDriverName =
```

Comment:
We don't need impala for hdi35, it's Cloudera only.

Reply:
Done!